CHANGELOG.md

```diff
@@ -38,7 +40,7 @@
   }
```

Comment:
could we also change it to `// handle error` to be consistent with previous code snippet?